native/explorer/src/dataframe/io.rs

```diff
@@ -356,11 +370,14 @@ pub fn df_from_ipc_stream(
 ) -> Result<ExDataFrame, ExplorerError> {
     let file = File::open(filename)?;
     let buf_reader = BufReader::new(file);
-    let df = IpcStreamReader::new(buf_reader)
+    let mut df = IpcStreamReader::new(buf_reader)
         .with_columns(columns)
         .with_projection(projection)
         .finish()?;
-    Ok(ExDataFrame::new(df))
+
+    let normalized_df = normalize_numeric_dtypes(&mut df)?;
+
+    Ok(ExDataFrame::new(normalized_df))
```

Comment:
Could this part be factored out into a function by using the `SerReader` trait so that future changes to the import throu of SerReader do not miss any of the functions?

Reply:
Thanks for the suggestion! I'm not sure if I understood correctly, but I added a function to abstract the "finish" process, which is more clear now. Please check again :)

Reply:
I'm merging this in order to add to the new release. Please fell free to open a PR if you think this can be improved.